attn_gradcheck.py

```diff
@@ -0,0 +1,54 @@
+import torch
+
+class Attn(torch.autograd.Function):
+    @staticmethod
+    def forward(ctx, q, k, v):
+        assert q.size() == k.size()
+        assert q.size()[0] == v.size()[0]
```

Comment:
Question [on the lab](https://github.com/pytorch/pytorch/wiki/Autograd-Onboarding-Lab) actually: are q, k, v defined to be 2D tensors? If so then these should be checks that they're all two dimensional